File: documentation/user/en/use/api/example/delete-entities-by-query.cs
evita.UpdateCatalog(
	"evita", session => {
		return session.DeleteEntities(
			Query(
				Collection("Brand"),
				FilterBy(
					And(
						AttributeStartsWith("name", "A")
					)
				),
				Require(
					Page(1, 20)
				)
			)
		);
	}
);

File: documentation/user/en/use/api/example/delete-entities-by-query.java
evita.updateCatalog(
	"evita", session -> {
		return session.deleteEntities(
			query(
				collection("Brand"),
				filterBy(
					and(
						attributeStartsWith("name", "A")
					)
				),
				require(
					page(1, 20)
				)
			)
		);
	}
);
